app/lib/methods/subscriptions/room.ts

```diff
@@ -69,6 +69,12 @@ export default class RoomSubscription {
 		}
 
 		reduxStore.dispatch(subscribeRoom(this.rid));
+		const rooms = reduxStore.getState().room;
+		if (rooms.rooms.length > 1) {
+			for (let i = 1; i < rooms.rooms.length; i++) {
+				reduxStore.dispatch(unsubscribeRoom(rooms.rooms[i]));
+			}
+		}
```

Comment:
If this solves a problem only for tablets then it should be wrapped around an if validation that checks the current device platform (e.g. `if (isTablet)`). We already count with `isTablet`, `isAndroid` and `isiOS`